src/MyMessage/index.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { useState } from 'react';
+
+import { Props } from './props';
+import { styles } from './styles';
+
+import { Dot } from '../Dot';
+
+import { isImage, getFileName } from '../util/file';
+import { formatTime, getDateTime } from '../util/dateTime';
+
+import { Row, Col, setConfiguration } from 'react-grid-system';
+
+setConfiguration({ maxScreenClass: 'xl' });
+
+export const MyMessage: React.FC<Props> = ({
```

Comment:
thinking ahead to `TheirMessage` - I'm thinking that there will be some logic that can be shared between the two e.g. `renderFiles`, `renderImages`, `renderReads`

idk how it's done in typescript but in Java we would use an Abstract Class for a base `Message` class then have `MyMessage` and `TheirMessage` extend it

at the very least they can be helper functions that both classes have access to

Reply:
Yeah and good catch, the new `ImageThumb` and `FileThumb` are refactored in now.

I'm going to pick this apart a bit more tomorrow, I think it can be a bit better.